### ISSUE TITLE
test with IP range results in failure

### ISSUE THREAD

**Issue by electrical** [mozilla/ssh_scan#499]
Hi, when running a test with the current docker container against an ip range i get the following error:

```
docker run -it mozilla/ssh_scan /app/bin/ssh_scan -t xx.xx.xx.0/25
```

```
Traceback (most recent call last):
	3: from /app/bin/ssh_scan:240:in `<main>'
	2: from /app/lib/ssh_scan/scan_engine.rb:197:in `scan'
	1: from /app/lib/ssh_scan/scan_engine.rb:197:in `each'
/app/lib/ssh_scan/scan_engine.rb:201:in `block in scan': undefined method `values' for []:Array (NoMethodError)
```

**Comment by claudijd** [comment]
@electrical thanks for the report!

**Comment by electrical** [comment]
I quickly validated the change and works! thank you for the quick followup on this.

**Comment by claudijd** [comment]
merged the fix into master, I'll try to get a gem release out this morning.  Thanks again to you both for the help here!

**Comment by claudijd** [comment]
I just released gem version 0.0.41, it should also be fixed in master and in the docker flow.  Again, many thanks for the help here!

**Comment by electrical** [comment]
Thank you :+1:
